new-frontiers-14/frontier-station-14
# Adds a bluespace vial

This PR adds a bluespace vial to the civilian research tree and corresponding fabrication recipes for the medical and science techfabs.




## About the PR



The bluespace vial holds 300 units, which seems to follow the 10x rule of bluespace containers. Given scaling this leaves it as less space efficient than the bluespace beaker but with alternative access to the centrifuge machine. So it occupies its own niche.

The production costs were meant to bridge somewhere between the bluespace syringe and beaker.

The default icon has some small alterations. Such as a metallic stopper and bluespace effects on the rim. This allows the vial to be differentiated from the standard while still retaining its fill components. I retained the standard vial components in this upload to keep everything in one place.

The vial is technically a baseitem as the standard vial has a breakable component which I felt was undesirable for a bluespace entity.

I tried my best to follow naming conventions. I recognize that some alterations may be required. The yml for the entity itself may have some baggage that I lack the skill to properly identify and remove. Though I did my best to prune it.

## Why / Balance


It felt like a niche that was missing. I feel that this will be desirable for fulfilling the role of centrifuging reagents in larger batches while serving as an attractive alternative to the blue space beaker for some reagent transport options. 

## How to test


Research vial on the civilian services track. It should be a tier 3 recipe.
Print from both the science and medical techfab.
Carry in both hands to ensure visuals are good to go.
Test fill states by filling with a reagent. Blood is ideal.
Centrifuge blood. 
Place vial in chemmaster to observe that blood has centrifuged.

Attempt to destroy the vial through throwing, explosions, harsh language. It should endure.

Throw vial. Contents should spill.

## Media


![ss+(2024-09-01+at+01 20 37)](https://github.com/user-attachments/assets/a0491dfc-3ef3-48fc-860c-9cae08e94fd5)
![ss+(2024-09-01+at+01 29 54)](https://github.com/user-attachments/assets/3caef6f0-ac32-4fe9-b8d1-6cf17c5146dd)



## Breaking changes


Ideally none.

**Changelog**
:cl: dustylens
- add: Added Bluespace vial, available in protolathe and medical techfab after T3 civilian research.
